src/js/extensions/auto-link.js

```diff
@@ -205,7 +205,7 @@ LINK_REGEXP_TEXT =
 
         createAutoLink: function (textNodes, href) {
             href = Util.ensureUrlHasProtocol(href);
-            var anchor = Util.createLink(this.document, textNodes, href),
+            var anchor = Util.createLink(this.document, textNodes, href, this.getEditorOption('targetBlank')),
```

Comment:
Also, it looks like both the auto-link extension and `Core.createLink` call `Util.createLink`, so I guess it shouldn't be moved out of Util...and maybe it does make sense to be called this...have to think on it.